src/dodal/devices/mx_phase1/beamstop.py

```diff
@@ -28,6 +30,7 @@ class BeamstopPositions(StrictEnum):
     """
 
     DATA_COLLECTION = "Data Collection"
+    OUT = "Out"
```

Comment:
Nit: For the aperture/scatterguard we've used `OUT_OF_BEAM` might be good to use the same convention here

Reply:
Yh, I'd be up for a common set if we can think of ones that work.